packages/types/src/procaptcha/manager.ts

```diff
@@ -78,3 +78,17 @@ export interface ProcaptchaState {
  * variables.
  */
 export type ProcaptchaStateUpdateFn = (state: Partial<ProcaptchaState>) => void
+
+export type Hash = string | number[]
```

Comment:
why are these not in tyeps?

Reply:
This is in types - moved to provider types to resolve below